packages/api/lambdas/bulk-operation.js

```diff
@@ -52,7 +52,10 @@ async function getGranuleIdsForPayload(payload) {
       body.hits.hits.forEach((hit) => {
         granuleIds.push(hit._source.granuleId);
       });
-      if (body.hits.total.value !== granuleIds.length) {
+
+      const totalHits = body.hits.total.value || body.hits.total;
```

Comment:
No issue with the change, but what is the cause? Can Elasticsearch return a response in multiple formats?

Reply:
Yeah so this was causing an infinite loop when I was testing the query payload. I didn't dig too deep into it, so maybe there's just some sort of ElasticSearch setting that needs to be changed for testing, but yes. The test was returning `body.hits.total` as a number rather than an object. When looking through ES documentation, this seems to be a valid response, though it wasn't clear to me when it would return one over the other (again, could be some setting we have 🤷🏻‍♀️). I figured it was a safe enough change